kubejs/server_scripts/base/recipes/minecraft/shaped.js

```diff
@@ -0,0 +1,43 @@
+ServerEvents.recipes(event => {
+    const id_prefex = 'enigmatica:base/minecraft/shaped/';
+
+    const recipes = [
+        {
+            output: '16x minecraft:stick',
+            pattern: ['A', 'A'],
+            key: {
+                A: '#minecraft:logs'
+            },
+            id: `${id_prefex}stick_16`
+        },
+        {
+            output: '4x minecraft:chest',
```

Comment:
no polymorph yet, so this might just override recipes

Reply:
removed. Will add in better stick recipe when wood registry is complete